src/components/ServerConnections.js

```diff
@@ -33,6 +33,7 @@ class ServerConnections extends ConnectionManager {
     constructor() {
         super(...arguments);
         this.localApiClient = null;
+        this.firstConnection = null;
```

Comment:
Maybe we should use a react context??? :man_shrugging:

Reply:
This value is pretty much single-use (it's set to null after fetching), so it might not benefit much from being turned into a react context?

Reply:
:+1: I was just trying to think of a more "react" way of handling this, but it's fine.